Source Main 5.2/source/WSclient.cpp

```diff
@@ -5417,7 +5417,7 @@ BOOL ReceiveDieExpLarge(const BYTE* ReceiveBuffer, BOOL bEncrypted)
     }
     else
     {
-        g_pMainFrame->SetPreExp(CharacterAttribute->Experience);
```

Comment:
encounters exception due to data type size mismatch (?). please verify if this is the correct fix, or we need to change the data type of Experience to just DWORD instead